### PR TITLE
refactor(components.api): move to webUniverseComponents

### PR DESCRIPTION
**Pull request by cbourgois**
### Description of the Change

08c0e04 - refactor(components.api): move to webUniverseComponents

ref: OM-148

### Related PR

- [x] https://github.com/ovh-ux/ovh-module-exchange/pull/183

/cc @jleveugle @frenautvh @antleblanc

### CODE REVIEW

File: client/app/webUniverseComponents/api/Api.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import angular from 'angular';
+
+export default /* @ngInject */ ($http, $q) => {
+  const self = this;
+
+  // DEPRECATED: You should use ovh.http
+  this.operation = (opt) => {
+    const requestUrl = URI.expand(opt.url, opt.urlParams || {})
```
Comment (antleblanc) [inline client/app/webUniverseComponents/api/Api.js]: Missing `URI` from module dependency right?

File: client/app/hosting/order/hosting-order.service.js [hunk 2]
```diff
@@ -3,17 +3,17 @@ angular.module('services').service(
   class HostingOrder {
     /**
      * Constructor
-     * @param Api
+     * @param WucApi
      * @param constants
      */
-    constructor(Api, constants) {
-      this.Api = Api;
+    constructor(WucApi, constants) {
```
Comment (antleblanc) [inline client/app/hosting/order/hosting-order.service.js]: Are we sure that `constants` will be available from the current module?

Comment (cbourgois) [reply client/app/hosting/order/hosting-order.service.js]: Good catch

Comment (cbourgois) [reply client/app/hosting/order/hosting-order.service.js]: Oh, `HostingOrder` is in manager, not in `webUniverseComponents`

**Comment by jleveugle** [comment]
Hello @cbourgois,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

**Pull request merged**